bundles/org.openhab.binding.homeconnect/src/main/resources/OH-INF/thing/thing-types.xml

```diff
@@ -481,6 +490,76 @@
 		</state>
 		<autoUpdatePolicy>veto</autoUpdatePolicy>
 	</channel-type>
+	<channel-type id="laundry_care_washer_idos1" advanced="true">
+		<item-type>Switch</item-type>
+		<label>i-Dos 1</label>
+		<description>This status indicates whether i-Dos 1 is activated for a washer program of the home appliance. (If
+			appliance
+			supports i-Dos)</description>
```

Comment:
Formatting looks curious. Can you put in in one line and apply spotless afterwards?

Reply:
Fixed